memcached.h

```diff
@@ -245,6 +245,12 @@ enum stop_reasons {
     EXIT_NORMALLY
 };
 
+enum close_reasons {
+    NORMAL_CLOSE,
+    ERROR_CLOSE,
```

Comment:
There might be value in making this more granular, maybe: server error, client/protocol error, others?

Reply:
looks like without making it more granular, it should be the default? all of these close_reason = lines seem redundant.